taskcluster/ci/build/kind.yml

```diff
@@ -51,6 +54,13 @@ job-defaults:
                     - ':{component}:lint'
                     - ':{component}:publish'
                     - ':{component}:renameSnapshotArtifacts'
+                nightly-release:
+                    - '-PnightlyVersion={nightlyVersion}'
```

Comment:
I would've like to avoid camelCase here but wasn't sure if gradle allows me to use `nightly-version` or `nightly_version`.

Reply:
`nightlyVersion` looks good! It's compliant with the other gradle commands 👍